envs/example/ci-full/group_vars/all.yml

```diff
@@ -16,13 +29,46 @@ neutron:
   lbaas:
     enabled: True
 
+common:
```

Comment:
I didn't mean to change ci-full env, until the CI failed at the same place for several time.
But my local test didn't fail. The difference between the CI job and my local test is that, I test by using ci-full-ubuntu, while the CI jobs use ci-full. Then I changed the ci-full env based on the ci-full-ubuntu env.